ShestakUI/Modules/Skins/Blizzard/EncounterJournal.lua

```diff
@@ -111,6 +111,13 @@ local function LoadSkin()
 	T.SkinScrollBar(EncounterJournalEncounterFrameInstanceFrameLoreScrollFrameScrollBar)
 	T.SkinScrollBar(EncounterJournalEncounterFrameInfoOverviewScrollFrameScrollBar)
 	T.SkinScrollBar(EncounterJournalEncounterFrameInfoBossesScrollFrameScrollBar)
+	
+	EncounterJournalSuggestFrame.Suggestion1.reward:StripTextures()
+	EncounterJournalSuggestFrame.Suggestion1.reward:SetTemplate("Transparent")
```

Comment:
Oh that's cool!

I tried it though but for some reason it distorts the icon:

https://s4.postimg.org/sik4iw019/Shestak.png